src/quic/bindingdata.h

```diff
@@ -26,6 +28,9 @@ enum class Side {
 };
 
 constexpr size_t kDefaultMaxPacketLength = NGTCP2_MAX_UDP_PAYLOAD_SIZE;
+constexpr size_t kMaxSizeT = std::numeric_limits<size_t>::max();
+constexpr int64_t kMaxSafeJsInteger = 9007199254740991;
```

Comment:
```suggestion
constexpr uint64_t kMaxSafeJsInteger = 9007199254740991u;
```

Looks like this is never used with signed semantics.